### PR TITLE
Flexo localization and genstrings resolution

### PR DESCRIPTION
**Pull request by jdizoglio**
removing genstrings warnings for different comments for key string

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdizoglio has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m [hunk 1]
```diff
@@ -479,7 +479,7 @@ - (void)switchToEmptyStateWithError:(NSError *)error
     if (error == nil) {
         errorMessage = NSLocalizedString(@"This folder is empty.", @"Label: Label displayed when the current folder is empty");
     } else {
-        errorMessage = NSLocalizedString(@"Unable to load contents of folder.", @"Label: Label displayed when the current folder is empty");
+        errorMessage = NSLocalizedString(@"Unable to load contents of folder.", @"Error message shown when a folder's contents could not be loaded.");
```
Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m]: We'd usually preface something like this with Label:

**Comment by boxtcarpel** [comment]
+2 LGTM

:information_source:

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m (hunk 1 above)
Comment (jdizoglio) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m]: Using our Message: comment prefix.

**Pull request merged**